src/docker/tools/index.go

```diff
@@ -103,11 +103,23 @@ func PullImage(c *client.Client, image string) error {
 	return err
 }
 
-// RunInDockerImage runs the given command in a new writeable container layer
+// RunInDockerContainer runs the given command in a new writeable container layer
 // over the given image, removes the container when the command exits, and returns
 // the output string and an error if any
-func RunInDockerImage(imageName, command string) (string, error) {
-	return util.Run("", fmt.Sprintf("docker run --rm %s %s", imageName, command))
+func RunInDockerContainer(runConfig RunConfig) error {
```

Comment:
Thoughts on creating `RunInDockerContainer` and `RunAndPipeInDockerContainer` so that the transform from runConfig to a command could be shared? I don't like the other uses of `util.Run` with `docker run`

Reply:
Hmm, I'm less convinced that's necessary because the two cases we would use `RunInDockerContainer` is 1) internally in `CatFile` and 2) for a test helper. Nothing in `runConfig` is set except imageName and command. It seems to me to be creating more complexity than needed. Plus `RunInDockerContainer` would be using `util.Run` as well anyways.

Reply:
K